crates/rpc-types-eth/src/transaction/request.rs

```diff
@@ -135,6 +135,12 @@ pub struct TransactionRequest {
     pub authorization_list: Option<Vec<SignedAuthorization>>,
 }
 
+macro_rules! tx_err {
+    ($self:expr, $msg:expr) => {
+        ValueError::new($self.clone(), $msg)
+    };
+}
```

Comment:
hmm, this is a bit problematic because this now always does a clone when used in `ok_or(`